.pre-commit-config.yaml

```diff
@@ -27,7 +33,7 @@ repos:
     rev: "v0.0.291"
     hooks:
       - id: ruff
-        args: [--fix, --exit-non-zero-on-fix]
+        args: ["--fix"]
```

Comment:
this is already handled by pre-commit as any change because of `--fix` yield in error